packages/vscode-extension/react-app/src/App.css

```diff
@@ -0,0 +1,28 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 80px;
+}
+
+.App-header {
+  background-color: #222;
+  height: 150px;
+  padding: 20px;
+  color: white;
+}
+
+.App-title {
+  font-size: 1.5em;
+}
+
+.App-intro {
+  font-size: large;
+}
+
+@keyframes App-logo-spin {
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
```

Comment:
Seems like you may be still in the process of cleaning things up, but I know `create-react-app` tends to generate a lot of cruft. Might be worth going through the `react-app` package and cleaning up anything unused.